cfme/tests/webui/test_splitter.py

```diff
@@ -57,6 +49,9 @@ def test_pull_splitter_persistence(request, appliance, model_object, destination
         assignee: anikifor
         caseimportance: low
         initialEstimate: 1/20h
+
+    Bugzilla:
+        1380443
```

Comment:
I don't think this BZ is related. Neither QE nor dev were able to reproduce it. Please remove it

Reply:
@lina-nikiforova:
1. I am not even checked `BZ` status.
2. I would like to collect `BZ` related info in test description. It help other (at least me) when we took new `FA` and start executing test; BZ collection related to that test help us to understand what was old owner testing way for that test (steps, non-fictional observations etc...)

Reply:
@digitronik ok, if it helps, I think it makes sense